plugins/home/src/homePageComponents/QuickStart/QuickStart.stories.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { QuickStartCard } from '../../plugin';
+import React, { ComponentType, PropsWithChildren } from 'react';
+import { wrapInTestApp, TestApiProvider } from '@backstage/test-utils';
+import Grid from '@material-ui/core/Grid';
+import ContentImage from './static/backstageSystemModel.png';
+
+export default {
+  title: 'Plugins/Home/Components/QuickStartCard',
+  decorators: [
+    (Story: ComponentType<PropsWithChildren<{}>>) =>
+      wrapInTestApp(
+        <TestApiProvider apis={[]}>
+          <Story />
+        </TestApiProvider>,
```

Comment:
if you are not using any apis I thiiink we should be able to just get rid of this wrapper, can you try that?

Reply:
looks like we still need `wrapInTestApp`. Without it we get a "missing context provider" error